apis/scheduling/v1alpha1/reservation_types.go

```diff
@@ -57,9 +57,9 @@ type ReservationSpec struct {
 	// reservation would be waiting to be available until free resources are sufficient.
 	// +optional
 	PreAllocation bool `json:"preAllocation,omitempty"`
-	// By default, reserved resources are always allocatable as long as the reservation phase is Available. When
```

Comment:
if AllocateOnce, the unused reservation resource (i.e. reservation.allocatable-reservation.allocated) should be ignored when schedule unmatched pod. this logic is already exists?

Reply:
yes. already exists